raiden/transfer/mediated_transfer/state.py

```diff
@@ -91,6 +92,7 @@ class LockedTransferSignedState(LockedTransferState):
     route_states: List[RouteState]
     balance_proof: BalanceProofSignedState = field(repr=False)
     metadata: Optional[Dict[str, Any]] = field(repr=False, default=None)
+    secret: Secret = field(default_factory=lambda: Secret(b""))
```

Comment:
I think we wanted to make sending the secret optional (which is enabled by default). Wouldn't be `None` as the default clearer? In general `b""` is a valid secret isnt it?